site/content/en/latest/api/gateway_api/backendtlspolicy.md

```diff
@@ -5,7 +5,7 @@ title = "BackendTLSPolicy"
 
     The `BackendTLSPolicy` resource is Alpha and has been part of
     the Experimental Channel since `v1.0.0`. For more information on release
-    channels, refer to our [versioning guide](https://gateway-api.sigs.k8s.io/concepts/versioning).
+    channels, refer to our [versioning guide](../concepts/versioning.md).
```

Comment:
The links in the generated gateway api docs are not correct. They should point to the Gateway API website.